### PR TITLE
[NO GBP] Crayon fixes

### PR DESCRIPTION
**Pull request by Helg2**
## About The Pull Request
I left `bite_consumption` be `charges / 5` which led to -1 being depleted on 5 and not giving any reagents on bite.
Also i somehow managed to change the total volume of crayons from 50 to 30 for usual ones and 100 for infinite. I'm a bit confused if number 50 was intended even. I tested it on the older branch and it was 30 and 100, so i dunno.
`override_infinity` is used just so you can't infinitely chew infinite crayons.
## Why It's Good For The Game
Stuff should work like intended.
## Changelog
:cl:
fix: eating mime's and rainbow crayons now properly transfers reagents.
fix: I somehow fixed the amount of reagents in crayons to what it was before.
/:cl:

### CODE REVIEW

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Looks fine, but could do with more clarity for future people who will touch this file.

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -211,16 +211,16 @@
...
 		AddComponent(/datum/component/palette, AVAILABLE_SPRAYCAN_SPACE, paint_color)
 
 	refill()
+	if(edible)
+		AddComponent(/datum/component/edible, bite_consumption = reagents.total_volume / (charges_left / 5), after_eat = CALLBACK(src, PROC_REF(after_eat)))
 
 /obj/item/toy/crayon/proc/after_eat(mob/user)
-	use_charges(user, 5, FALSE)
-	if(check_empty(user)) //Prevents division by zero
+	use_charges(user, 5, FALSE, TRUE)
```
[the remark this answers is not in the document]
Comment (Helg2) [reply code/game/objects/items/crayons.dm]: I don't get how, but it somehow breaks linters

[the remark this answers is not in the document]
Comment (Jacquerel) [reply code/game/objects/items/crayons.dm:223]: Set it back the linter was actually informing you about an oversight we both missed.
That oversight is that you added an argument to `obj/item/toy/crayon/spraycan/proc/use_charges` but left `obj/item/toy/crayon/spraycan/borg/use_charges` without the new argument.

[the remark this answers is not in the document]
Comment (Helg2) [reply code/game/objects/items/crayons.dm:223]: Oh right

**Pull request merged**